cfg.go

```diff
@@ -61,9 +64,77 @@ func (c *configWrapper) mergeEnv(envName string) error {
 	return c.v.MergeInConfig()
 }
 
+func (c *configWrapper) loadEnvConfigFile() {
+	tempV := viper.New()
+
+	tempV.SetConfigType(c.params.FileType)
+	tempV.SetConfigName(envVarConfigFile)
+	tempV.AddConfigPath(c.params.Path)
+	if err := tempV.ReadInConfig(); err != nil {
+		return
+	}
+	keys := tempV.AllKeys()
+	for _, key := range keys {
+		envVarName := tempV.GetString(key)
+		if value := os.Getenv(envVarName); value != "" {
```

Comment:
We should use `os.LookupEnv` instead: in that way we can determine whether the variable doesn't exist or the value is "".